.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "libkineto/third_party/fmt"]
 	path = libkineto/third_party/fmt
 	url = https://github.com/fmtlib/fmt.git
+[submodule "libkineto/third_party/dynolog"]
```

Comment:
maybe minor - but why include dynolog in kineto instead of the other way around? There would be less to checkout when building pytorch.

Reply:
yea that is a fair point. Not a great answer but we are maintaining ipcfabric inside dynolog project. There are some in-house use-cases for it outside of Kineto too. 
-  In terms of impact  on pytorch at least we are not building dyno files at all, we just need to use ipcfabric directory that is a header only library :)
- it sill will checkout the dynolog files however.
- i'll look into some way to do a shallow checkout on submodules.